### PR TITLE
Bookstore: create and remove books

### PR DESCRIPTION
**Pull request by aliabbani**
## Bookstore: create and remove books

In this milestone we:

- Created store using react-redux
- Wrote book's actions and reducer.
- Created new book object in Redux store, & dispatched add book and remove book functions, 
- Used the redux-logger to check if the application is working correctly

### CODE REVIEW

File: src/components/BookContainer.js [hunk 1]
```diff
@@ -1,31 +1,47 @@
...
+
+// const addBookItem = (title) => {
+//   const newBook = {
+//     id: uuidv4(),
+//     title,
+//   };
+//   setBookOri({
+//     books: [...BookOri.books, newBook],
+//   });
+// };
```
Comment (GzimAsani) [inline src/components/BookContainer.js]: - [x] Remove the commented code.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Required Changes :recycle: 

## Hello  @aliabbani    hope you're well :dove: You have done great job so far :100:  

## To highlight:
- [x] The gitflow has been followed :+1: 

## To improve:
- [x] Check inline comments for more information

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 You can also consider:

- [ ] Add a picture to your readme file, it'll look more professional :thought_balloon: 

 Cheers and Happy coding!

 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/BookContainer.js (hunk 1 above)
Comment (aliabbani) [reply src/components/BookContainer.js]: Thank you @GzimAsani, Thank you for your comment, I removed all the commented code

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @aliabbani, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**